Notes_and_Examples/5_CurrentFlow/5_currentDirection.xml

```diff
@@ -0,0 +1,108 @@
+<problem display_name="webGLDemo">
```

Comment:
@dseaton suggestion: `display_name` isn't something magical - it's just a part of edX default markup for custom JS problem; feel free to change it to better suit your needs.